geode-core/src/test/java/org/apache/geode/internal/cache/CacheOperationMessageTest.java

```diff
@@ -34,15 +34,15 @@ public void shouldBeMockable() throws Exception {
     ClusterDistributionManager mockDistributionManager = mock(ClusterDistributionManager.class);
 
     when(mockCacheOperationMessage.supportsDirectAck()).thenReturn(true);
-    when(mockCacheOperationMessage._mayAddToMultipleSerialGateways(eq(mockDistributionManager)))
+    when(mockCacheOperationMessage._mayAddToSerialGateway(eq(mockDistributionManager)))
```

Comment:
I realize it was previously named with an underscore, but is there a compelling reason not to just clean that up when renaming? Is the underscore supposed to be expressing something?

Reply:
Yes, I thought about that too. I think it was a way to differentiate the API name from the implementation name. There are two implementations of the '_' method that basically delegate to LocalRegion.notifiesSerialGateway. The LocalRegion method does all the real work, so actually the only change to address this JIRA needed to be in that method. I thought I would just rename the methods so they reflected the actual behavior, but I didn't take it far enough. I also missed the renames in the PR messages. I'll clean all that up. Thanks for the comment.